src/ImportComponent.js

```diff
@@ -0,0 +1,27 @@
+class ImportComponent {
```

Comment:
New class for the import functionality. Right now, this only offers a single static function `importNewComponent`, but keeping it in a class feels like a good way to organize the code going forward. We may rip this out in a future breaking change if this feels too restrictive.